wgsl/index.bs

```diff
@@ -7739,9 +7740,9 @@ Converts to:
 
 <div class='example wgsl function-scope' heading="For to Loop transformation">
   <xmp highlight='rust'>
+    var a: i32 = 2;
```

Comment:
The other loop examples used a value of `2` so I did the same here.

Reply:
However looking a bit more at the other loop examples the loop is a bit different so another value might make more sense. Make it back to 0, or -2 for example.

Reply:
The specific value doesn't really matter except that it might confuse readers.
The main thing is to have variables declared in the right place and get the scoping right.